### DISCUSSION TITLE
How can we use Typewind properly in a monorepo? (Nx as example)

### DISCUSSION THREAD

**Discussion by gilvex** [Mokshit06/typewind#40] [Q&A]
Hey I'm using Typewind and would like to find out how I can setup it to use it on nx monorepo 

I have 2 apps (Web on Next.js and Desktop on React + Vite) Both of them use tailwind but they both may have some alterations in their own config 

Both of their configs use tailwind.preset.js as a base And Tailwind generates types from that preset

**Comment by kavinvalli** [comment]
Hi! Does this help answer your question: https://github.com/Mokshit06/typewind/issues/18#issuecomment-1437842032?

**Reply by gilvex** [reply]
It does not because on nx monorepo we have only 1 node_modules and package.json and it is in the root

If we do that the build will fail
